R/PipeOpFFS.R

```diff
@@ -57,10 +60,7 @@ PipeOpFFS = R6Class("PipeOpFFS",
         drop = p_lgl(tags = c("train", "predict", "required")),
         left = p_dbl(tags = c("train", "predict", "required")),
         right = p_dbl(tags = c("train", "predict", "required")),
-        feature = p_fct(
-          levels = c("mean", "max", "min", "slope", "median", "var"),
-          tags = c("train", "predict", "required")
-        )
+        features = p_uty(tags = c("train", "predict", "required"))
```

Comment:
`ParamUty` has a `custom_check` argument. The validation that is done in `.transform` should be done here. 
(The case with left and right is a bit special, because there the relationship between two parameters is considered, here this is not the case).